templates/EmulatorConstants.sol.template

```diff
@@ -14,16 +14,17 @@
 // limitations under the License.
 //
 
-/// @title UArchConstants
+/// @title EmulatorConstants
 /// @notice Contains constants for micro-architecture
```

Comment:
I think the description should also change to reflect that it's no longer micro-architecture constants.